utils/utils.c

```diff
@@ -191,6 +191,25 @@ bool is_ancestor(char *path, char *data_path)
 	return (path[len] == '/' || path[len] == '\0');
 }
 
+bool is_uftrace_dir(char *path)
+{
+	int fd;
+	char *info_path = NULL;
+	char sig[8] = {0,};
+
+	// <uftrace data dir>/info always be exist.
+	xasprintf(&info_path, "%s/info", path);
+	fd = open(info_path, O_RDONLY);
+	if (fd == -1) {
+		return false;
+	}
+
+	// read string signature "Ftrace!"
```

Comment:
I don't want to expose this detail in the comment or commit logs.  It can be changed anytime..